components/button/button.html

```diff
@@ -51,14 +51,14 @@ <h2 class="comp-lib-h2">Primary Button</h2>
     <h2 class="comp-lib-h2">Link Button</h2>
     <div class="component_demo_container">
         <!-- Link Button Component Starts -- Link Primary -->
-        <a class="link link--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs" href="#">
+        <a class="dui-link dui-link--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs" href="#">
```

Comment:
if you can please refractor the number of classes

Reply:
Yup will be consolidating in further iterations.